src/Kernel32.Shared/Kernel32.Helpers.cs

```diff
@@ -47,5 +48,60 @@ public static Win32ErrorCode GetLastError()
         {
             return (Win32ErrorCode)Marshal.GetLastWin32Error();
         }
+
```

Comment:
I wondered about that, the DllImport have extensive documentation as we get it from MSDN but matching such details in every helper method variant would be painful. Maybe i'll keep to a minimal documentation of what the method and parameter do, not all failure cases and possibilities like the DllImport ones.